### PR TITLE
[CI] Add no-HMAT qemu configuration files

### PR DESCRIPTION
**Pull request by KFilipek**
Depends on #469 

### Checklist

- [x] Code compiles without errors locally
- [x] All tests pass locally
- [x] CI workflows execute properly

- [x] CI workflows, not executed per PR (e.g. Nightly), execute properly  [here](https://github.com/KFilipek/unified-memory-framework/actions/runs/9095559817)
- [ ] New tests added, especially if they will fail without my changes
- [ ] Added/extended example(s) to cover this functionality
- [ ] Extended the README/documentation
- [x] All newly added source files have a license
- [ ] All newly added source files are referenced in CMake files
- [ ] Logger (with debug/info/... messages) is used

### CODE REVIEW

File: scripts/qemu/configs/sock_2_var2_hmat.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+# Copyright (C) 2021 - 2024 Intel Corporation.
+-->
+
+<!--
+
+Total Sockets: 2
+Total CPUs: 4
+Thread per CPU: 1
+Total Memory: 4096MiB
+
```
Comment (lukaszstolarczuk) [inline scripts/qemu/configs/sock_2_var2_hmat.xml:11]: missied HMAT param here

**Review by lukaszstolarczuk — COMMENTED** [review]
👍

File: scripts/qemu/configs/sock_2_var2_hmat.xml (hunk 1 above)
Comment (KFilipek) [reply scripts/qemu/configs/sock_2_var2_hmat.xml:11]: Done

Comment (KFilipek) [reply scripts/qemu/configs/sock_2_var2_hmat.xml]: Done

File: scripts/qemu/configs/sock_4_var1_hmat.xml [hunk 2]
```diff
@@ -2,6 +2,26 @@
 # Copyright (C) 2020 - 2024 Intel Corporation.
 -->
 
+<!--
+
+Total Sockets: 4
+Total CPUs: 4
+Thread per CPU: 1
+Total Memory: 10240MiB
+HMAT: Disabled
```
[the remark this answers is not in the document]
Comment (KFilipek) [reply scripts/qemu/configs/sock_4_var1_hmat.xml:11]: Oh, right

**Review by lukaszstolarczuk — APPROVED** [review]

**Comment by lukaszstolarczuk** [comment]
pls rebase to make CI green

**Review by PatKamin — APPROVED** [review]

**Review by bratpiorka — APPROVED** [review]

**Comment by KFilipek** [comment]
> pls rebase to make CI green

Done, I'm waiting for the CI to do the job.

**Pull request merged**